var/spack/repos/builtin/packages/xdmf3/package.py

```diff
@@ -13,32 +18,63 @@ class Xdmf3(CMakePackage):
     """
 
     homepage = "http://xdmf.org"
-    url      = "https://gitlab.kitware.com/xdmf/xdmf/-/archive/master/xdmf-master.tar.bz2"
-    git      = "https://gitlab.kitware.com/xdmf/xdmf.git"
+    git = "https://gitlab.kitware.com/xdmf/xdmf.git"
 
     # There is no official release of XDMF and development has largely ceased,
     # but the current version, 3.x, is maintained on the master branch.
-    version('2019-01-14', commit='8d9c98081d89ac77a132d56bc8bef53581db4078')
+    version("2019-01-14", commit="8d9c98081d89ac77a132d56bc8bef53581db4078")
+
+    variant("shared", default=True, description="Enable shared libraries")
+    variant("mpi", default=True, description="Enable MPI")
 
-    variant('shared', default=True, description='Enable shared libraries')
-    variant('mpi', default=True, description='Enable MPI')
+    depends_on("libxml2")
+    depends_on("boost")
+    depends_on("libtiff")
+    depends_on("mpi", when="+mpi")
+    depends_on("hdf5+mpi", when="+mpi")
+    depends_on("hdf5~mpi", when="~mpi")
 
-    depends_on('libxml2')
-    depends_on('boost')
-    depends_on('mpi', when='+mpi')
-    depends_on('hdf5+mpi', when='+mpi')
-    depends_on('hdf5~mpi', when='~mpi')
+    patch("fix-libs.patch")
 
     def cmake_args(self):
         """Populate cmake arguments for XDMF."""
         spec = self.spec
 
         cmake_args = [
-            '-DBUILD_SHARED_LIBS=%s' % str('+shared' in spec),
-            '-DXDMF_BUILD_UTILS=ON',
-            '-DXDMF_WRAP_JAVA=OFF',
-            '-DXDMF_WRAP_PYTHON=OFF',
-            '-DXDMF_BUILD_TESTING=ON'
+            "-DBUILD_SHARED_LIBS=%s" % str("+shared" in spec),
+            "-DXDMF_BUILD_UTILS=ON",
+            "-DXDMF_WRAP_JAVA=OFF",
+            "-DXDMF_WRAP_PYTHON=OFF",
+            "-DXDMF_BUILD_TESTING=OFF",
         ]
 
         return cmake_args
+
+    def install(self, spec, prefix):
+        super().install(spec, prefix)
```

Comment:
This `super()` won't work in Python 2 as far as I know. You're better off doing:
```python
@run_after('install')
def post_install(self):
    ...
```